### PR TITLE
Update on index.tmpl

### LINKED ISSUE #32: Banner always shows "Containerlab Topology: nokia-MAGc-lab"
**Description**
The containerlab topology information always shows "nokia-MAGc-lab" instead of the topology name in clab-topo.yaml file.
Clab version: 0.44.3
Topoviewer version: nightly-23.10.22b

**CLAB Topology yaml**
```
name: dharmapu-clab-2nodes
prefix: __lab-name
mgmt:
  network: net_dharmapu-clab-2nodes
  ipv4-gw: 172.36.36.254
  ipv4-subnet: 172.36.36.0/24
  bridge: brmp-dharmapu
topology:
  nodes:
    R1:
      kind: vr-sros
      image: registry.srlinux.dev/pub/vr-sros:23.7.R1
      type: cpu=2 min_ram=4 max_nics=6 chassis=sr-1 slot=A card=cpm-1 slot=1 card=iom-1 mda/1=me6-100gb-qsfp28
      license: /clab/license/timos-latest.txt
      mgmt-ipv4: 172.36.36.1
      startup-config: configs/R1_R1_Core.cfg
      group: CORE
    R2:
      kind: vr-sros
      image: registry.srlinux.dev/pub/vr-sros:23.7.R1
      type: 'cp: cpu=2 min_ram=4 chassis=ixr-e slot=A card=cpm-ixr-e/imm14-10g-sfp++4-1g-tx ___ lc: cpu=2 min_ram=4 max_nics=18 chassis=ixr-e slot=1 card=imm14-10g-sfp++4-1g-tx  mda/1=m14-10g-sfp++4-1g-tx'
      license: /clab/license/timos-latest.txt
      mgmt-ipv4: 172.36.36.2
      startup-config: configs/R2_R2_Core.cfg
      group: CORE
    topoviewer:
      kind: linux
      mgmt-ipv4: 172.36.36.253
      image: ghcr.io/asadarafat/topoviewer:nightly-23.10.22b
      ports:
      - 8081:8081
      startup-delay: 3
      entrypoint: /opt/topoviewer/topoviewer clab --allowed-hostnames 192.168.10.16 --server-port 8081 --topology-file-json local-bind/topo-file.json
      binds:
      - /clab/dharmapu/clab-dharmapu-clab-2nodes/topology-data.json:/opt/topoviewer/local-bind/topo-file.json:ro
  links:
  - endpoints:
    - R1:eth1
    - R2:eth1
```

### LINKED ISSUE #33: Topology or namespace name always has "clab" prefix in clab capture string when starting wireshark
**Description**
When I use below clab-topo.yaml file, the generated clab-capture string is `clab-capture://root@192.168.10.16:8081/?clab-dharmapu-clab-2nodes-R1?eth1`. The topology name always has `clab-` prefix. The expected topology name is `dharmapu-clab-2nodes-R1`.

**Environment**
- User deploy the topology to clab server without using root user
- User deploy the topology to clab server using username the same name as windows user
- User uses Windows 10 OS

**IP NETNS result in server**
```
/clab/dharmapu$ sudo ip netns
dharmapu-clab-2nodes-topoviewer
dharmapu-clab-2nodes-R1 (id: 1)
dharmapu-clab-2nodes-R2 (id: 0)
```

**CLAB Topology yaml**
```
name: dharmapu-clab-2nodes
prefix: __lab-name
mgmt:
  network: net_dharmapu-clab-2nodes
  ipv4-gw: 172.36.36.254
  ipv4-subnet: 172.36.36.0/24
  bridge: brmp-dharmapu
topology:
  nodes:
    R1:
      kind: vr-sros
      image: registry.srlinux.dev/pub/vr-sros:23.7.R1
      type: cpu=2 min_ram=4 max_nics=6 chassis=sr-1 slot=A card=cpm-1 slot=1 card=iom-1 mda/1=me6-100gb-qsfp28
      license: /clab/license/timos-latest.txt
      mgmt-ipv4: 172.36.36.1
      startup-config: configs/R1_R1_Core.cfg
      group: CORE
    R2:
      kind: vr-sros
      image: registry.srlinux.dev/pub/vr-sros:23.7.R1
      type: 'cp: cpu=2 min_ram=4 chassis=ixr-e slot=A card=cpm-ixr-e/imm14-10g-sfp++4-1g-tx ___ lc: cpu=2 min_ram=4 max_nics=18 chassis=ixr-e slot=1 card=imm14-10g-sfp++4-1g-tx  mda/1=m14-10g-sfp++4-1g-tx'
      license: /clab/license/timos-latest.txt
      mgmt-ipv4: 172.36.36.2
      startup-config: configs/R2_R2_Core.cfg
      group: CORE
    topoviewer:
      kind: linux
      mgmt-ipv4: 172.36.36.253
      image: ghcr.io/asadarafat/topoviewer:nightly-23.10.22b
      ports:
      - 8081:8081
      startup-delay: 3
      entrypoint: /opt/topoviewer/topoviewer clab --allowed-hostnames 192.168.10.16 --server-port 8081 --topology-file-json local-bind/topo-file.json
      binds:
      - /clab/dharmapu/clab-dharmapu-clab-2nodes/topology-data.json:/opt/topoviewer/local-bind/topo-file.json:ro
  links:
  - endpoints:
    - R1:eth1
    - R2:eth1
```

### LINKED ISSUE #34: Link impairment does not work
**Description**
Link impairment seems not work. When I set delay 100000 ms, ping delay is still under 10ms.

**Environment**
- User deploy the topology to clab server without using root user
- User deploy the topology to clab server using username the same name as windows user
- User uses Windows 10 OS

**IP NETNS result in server**
```
/clab/dharmapu$ sudo ip netns
dharmapu-clab-2nodes-topoviewer
dharmapu-clab-2nodes-R1 (id: 1)
dharmapu-clab-2nodes-R2 (id: 0)
```

**CLAB Topology yaml**
```
name: dharmapu-clab-2nodes
prefix: __lab-name
mgmt:
  network: net_dharmapu-clab-2nodes
  ipv4-gw: 172.36.36.254
  ipv4-subnet: 172.36.36.0/24
  bridge: brmp-dharmapu
topology:
  nodes:
    R1:
      kind: vr-sros
      image: registry.srlinux.dev/pub/vr-sros:23.7.R1
      type: cpu=2 min_ram=4 max_nics=6 chassis=sr-1 slot=A card=cpm-1 slot=1 card=iom-1 mda/1=me6-100gb-qsfp28
      license: /clab/license/timos-latest.txt
      mgmt-ipv4: 172.36.36.1
      startup-config: configs/R1_R1_Core.cfg
      group: CORE
    R2:
      kind: vr-sros
      image: registry.srlinux.dev/pub/vr-sros:23.7.R1
      type: 'cp: cpu=2 min_ram=4 chassis=ixr-e slot=A card=cpm-ixr-e/imm14-10g-sfp++4-1g-tx ___ lc: cpu=2 min_ram=4 max_nics=18 chassis=ixr-e slot=1 card=imm14-10g-sfp++4-1g-tx  mda/1=m14-10g-sfp++4-1g-tx'
      license: /clab/license/timos-latest.txt
      mgmt-ipv4: 172.36.36.2
      startup-config: configs/R2_R2_Core.cfg
      group: CORE
    topoviewer:
      kind: linux
      mgmt-ipv4: 172.36.36.253
      image: ghcr.io/asadarafat/topoviewer:nightly-23.10.22b
      ports:
      - 8081:8081
      startup-delay: 3
      entrypoint: /opt/topoviewer/topoviewer clab --allowed-hostnames 192.168.10.16 --server-port 8081 --topology-file-json local-bind/topo-file.json
      binds:
      - /clab/dharmapu/clab-dharmapu-clab-2nodes/topology-data.json:/opt/topoviewer/local-bind/topo-file.json:ro
  links:
  - endpoints:
    - R1:eth1
    - R2:eth1
```

### PR DESCRIPTION
**Pull request by gusman**
- Fix #32, update the code as mentioned in issue description
- Fix #33, update thy cyto template by removing ${type} prefix
- Fix #34, by changing the approach for executing ssh command in clab server (host) to generate ssh command that can be copied by user on his terminal.

### CODE REVIEW

File: .github/workflows/docker-ci-v2.yml
Comment (asadarafat) [inline .github/workflows/docker-ci-v2.yml:1]: dont change this. keep it as reference. ci-v3 will be used

File: .github/workflows/docker-ci-v3.yml
Comment (asadarafat) [inline .github/workflows/docker-ci-v3.yml:1]: how `type=ref,event=tag w` will be used?

File: dist/html-static/clab-client/clab-client-mac/clab-capture.scpt
Comment (asadarafat) [inline dist/html-static/clab-client/clab-client-mac/clab-capture.scpt:1]: let's keep this file for reference

**Review by asadarafat — CHANGES_REQUESTED** [review]
please change the request after addressing the comments i gave.

File: dist/html-static/clab-client/clab-client-mac/clab-capture.scpt
Comment (gusman) [reply dist/html-static/clab-client/clab-client-mac/clab-capture.scpt:1]: I rebase the PR code to excluded changes in `./dist` directory

File: .github/workflows/docker-ci-v2.yml
Comment (gusman) [reply .github/workflows/docker-ci-v2.yml:1]: I have changed back `docker-ci-v2.yaml` to be same as `master`/`main` branch

File: .github/workflows/docker-ci-v3.yml
Comment (gusman) [reply .github/workflows/docker-ci-v3.yml:1]: By adding `type=ref,event=tag`, it allows me to build and publish the docker image by selecting `git tag` value,
`git tag` become the tag of docker image as well.

**Comment by gusman** [comment]
> please change the request after addressing the comments i gave.

I have addressed all the comments.
Need you review again.
If the all the changes are OK, please help to build and publish new docker image.

**Review by asadarafat — APPROVED** [review]

**Pull request merged**